x-pack/solutions/search/plugins/search_playground/public/components/query_mode/query_viewer.tsx

```diff
@@ -81,18 +126,39 @@ export const ElasticsearchQueryViewer = ({
           </EuiFlexItem>
         </EuiFlexGroup>
       </EuiSplitPanel.Inner>
+      {userElasticsearchQueryValidations?.isUserCustomized &&
+      (userElasticsearchQueryValidations?.userQueryErrors?.length ?? 0) > 0 ? (
+        <EuiSplitPanel.Inner grow={false}>
+          {userElasticsearchQueryValidations.userQueryErrors!.map((error, errorIndex) => (
+            <EuiCallOut
```

Comment:
I'd rather have this displayed in the editor. And I'm investigating the ability to do that, but do far that is looking more complicated than I'd like. so this is a stop-gap until I figure it out.